ldk/javascript/src/whisper/types.ts

```diff
@@ -611,6 +636,19 @@ export type Progress = WhisperComponent<WhisperComponentType.Progress> & {
   size?: StyleSize;
 };
 
+export type Grid = WhisperComponent<WhisperComponentType.Grid> & {
+  alignContent?: AlignContent;
+  alignItems?: AlignItems;
+  children: Array<ChildComponents>;
+  container: boolean;
+  direction?: GridDirection;
+  justifyContent?: JustifyContent;
+  item: boolean;
+  spacing?: number;
+  wrap?: Wrap;
+  xs?: string;
+};
```

Comment:
This would work for making `container` OR `item` required, making them mutually exclusive

```suggestion
export type Grid = WhisperComponent<WhisperComponentType.Grid> & {
  alignContent?: AlignContent;
  alignItems?: AlignItems;
  children: Array<ChildComponents>;
  direction?: GridDirection;
  justifyContent?: JustifyContent;
  spacing?: number;
  wrap?: Wrap;
  xs?: string;
} & (
  | {
      container: boolean
    }
  | {
      item: boolean
    }
);
```